### PR TITLE
Cosmos Hub and Meichain function binding

### LINKED ISSUE #29: Cosmos Hub - Show ATOM and USD Balance

### PR DESCRIPTION
**Pull request by tharamalai**

### CODE REVIEW

File: ui/src/contexts/MeichainContext.js [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+        memo: "",
+        account_number: String(data.result.value.account_number),
+        sequence: String(data.result.value.sequence)
+      });
+    
+      let signedTx = meichain.sign(stdSignMsg, privateKey);
+      signedTx = convertSignMsg(signedTx)
+      meichain.broadcast(signedTx).then(response => console.log(response))
+    })
+  }
```
Comment (defi-pizza) [inline ui/src/contexts/MeichainContext.js]: The sign and broadcast are duplicated. Make it as helper function.

File: ui/src/utils.js [hunk 2]
```diff
@@ -1,5 +1,191 @@
...
+  }
+  return host
+}
+
+export const getCosmosLcdUrl = () => {
+  const host = getHost()
+  if (host === "localhost") {
+    return "http://localhost:8012"
+  }
+  return "http://gaia-ibc-hackathon.node.bandchain.org:8000"
```
Comment (defi-pizza) [inline ui/src/utils.js]: Make `http://gaia-ibc-hackathon.node.bandchain.org` constant.

File: ui/src/utils.js [hunk 3]
```diff
@@ -1,5 +1,191 @@
...
+}
+
+export const getCosmosRestServer = () => {
+  const host = getHost()
+  if (host === "localhost") {
+    return "http://localhost:8011"
+  }
+  return "http://gaia-ibc-hackathon.node.bandchain.org:1317"
```
Comment (defi-pizza) [inline ui/src/utils.js]: Use constant instead.

File: ui/src/contexts/MeichainContext.js (hunk 1 above)
Comment (tharamalai) [reply ui/src/contexts/MeichainContext.js]: fixed

File: ui/src/utils.js (hunk 2 above)
Comment (tharamalai) [reply ui/src/utils.js]: fixed

File: ui/src/utils.js (hunk 3 above)
Comment (tharamalai) [reply ui/src/utils.js]: fixed

**Pull request merged**